coalib/results/Diff.py

```diff
@@ -352,6 +352,7 @@ def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
         """
+        line_nr = 0 if (len(self._file) < line_nr) else line_nr
```

Comment:
@RohanVB Only a single value had to be assigned using if-else so I used this format to keep it short and simple :smiley: